sunpy/net/dataretriever/tests/test_fermi_gbm.py

```diff
@@ -46,24 +97,30 @@ def test_can_handle_query(time):
     assert ans4 is False
 
 
-@pytest.mark.remote_data
-@pytest.mark.parametrize("time,instrument", [
-    (a.Time('2012/8/9', '2012/8/10'), a.Instrument('gbm')),
-])
-def test_query(time, instrument):
-    qr1 = LCClient.search(time, instrument)
+@mock.patch('sunpy.net.dataretriever.sources.fermi_gbm.GBMClient.search',
+            return_value=mock_querry_object('2012/8/9', '2012/8/10'))
+def test_query(mock_search):
+    qr1 = LCClient.search(a.Time('2012/8/9', '2012/8/10'),
+                          a.Instrument('GBM'))
     assert isinstance(qr1, QueryResponse)
     assert len(qr1) == 2
-    assert qr1.time_range().start == time.start
-    assert qr1.time_range().end == time.end
+    assert qr1.time_range().start == parse_time('2012/08/09')
+    assert qr1.time_range().end == parse_time('2012/08/10')
 
 
-@pytest.mark.remote_data
-@pytest.mark.parametrize("time,instrument", [
-    (a.Time('2012/11/27', '2012/11/27'), a.Instrument('gbm')),
-])
-def test_get(time, instrument):
-    qr1 = LCClient.search(time, instrument)
+# @pytest.mark.remote_data
+# @pytest.mark.parametrize("time,instrument", [
+#     (a.Time('2012/11/27', '2012/11/27'), a.Instrument('gbm')),
+# ])
+# def test_get(time, instrument):
+#     qr1 = LCClient.search(time, instrument)
+#     download_list = LCClient.fetch(qr1)
+#     assert len(download_list) == len(qr1)
+
+@mock.patch('sunpy.net.dataretriever.sources.fermi_gbm.GBMClient.search',
+            return_value=mock_querry_object('2012/11/27', '2012/11/27'))
+def test_get(time):
+    qr1 = LCClient.search(time, a.Instrument('gbm'))
```

Comment:
I'm not sure this test makes any sense as a mock test. But I don't have a better idea.

Reply:
what do you mean? Like theres no point in mocking this test?

Reply:
Oh I got the wrong test, when I was looking at the diff.